app/forms/select_schools_form.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class SelectSchoolsForm
+  include ActiveModel::Model
+  include ActiveModel::Serialization
+
+  attr_accessor :school_ids
+  attr_accessor :delivery_partner_id
+  attr_accessor :source
```

Comment:
We can define them all in one line, and we do it in other forms.

Reply:
Old habbit from reform/dry-validations :)